pilot/pkg/config/clusterregistry/conversion.go

```diff
@@ -180,44 +232,11 @@ func validateCluster(crPath string, cluster *k8s_cr.Cluster) (err error) {
 			err = multierror.Append(err, err1)
 		}
 	}
-
-	if cluster.ObjectMeta.Annotations[ClusterAccessConfigFile] == "" {
-		err = multierror.Append(err, fmt.Errorf("cluster %s doesn't have a valid config file", cluster.ObjectMeta.Name))
+	if cluster.ObjectMeta.Annotations[ClusterAccessConfigSecret] == "" {
```

Comment:
Isn't it fine (i.e. plain http ) ? Ok for testing.

Reply:
Secret here contains not just `certificate-authority-data` but also `server` key which should be used to access that specific API server, so having secret is mandatory even with http type of connection.